src/google/protobuf/util/internal/utility.cc

```diff
@@ -302,7 +302,7 @@ bool IsMap(const google::protobuf::Field& field,
 string DoubleAsString(double value) {
   if (value == std::numeric_limits<double>::infinity()) return "Infinity";
   if (value == -std::numeric_limits<double>::infinity()) return "-Infinity";
-  if (::isnan(value)) return "NaN";
+  if (std::isnan(value)) return "NaN";
```

Comment:
std::isnan is a C++11 feature. We probably need to stick with ::isnan. Why can't "::isnan" be used here?